pkg/filesystem/local_directory_windows.go

```diff
@@ -108,10 +108,34 @@ func newLocalDirectory(absPath string, openReparsePoint bool) (DirectoryCloser,
 	return newLocalDirectoryFromHandle(handle)
 }
 
+// Convert forward-slash drive-letter paths to absolute drive-letter paths.
+// This can come from how build directories are configured for bb-worker on Windows.
+// Where a combination of 'git-bash' and environment variable expansion in jsonnet can create paths
+// that the `filepath` does not detect as absolute.
+//
+// Example: /C:/...
+// Should be C:/...
+func CanonicalizeAbsolutePath(path string) string {
```

Comment:
I don't fully understand this part. `NewLocalDirectory()` be called with whatever is the native representation of a path on that operating system. So I guess that for Windows this means that you should just use drive names and backslashes as part of your config files...?

Reply:
No longer relevant